lib/src/current_weather/data/adapters/isar_current_weather_adapter.dart

```diff
@@ -0,0 +1,30 @@
+import '../../domain/entities/current_weather.dart';
+import '../models/isar_current_weather_dto.dart';
+
+extension CurrentWeatherToIsarCurrentWeatherAdapter on CurrentWeather {
+  IsarCurrentWeatherDto toDto() {
```

Comment:
About naming: could you name local - related classes with a `Local` prefix instead of a `Isar` prefix?

This means changing:
- This adapter's name
- This adapter's file name
- Its relative model class name, and its file name
- etc.